src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] arrayData = fileReader(fromFileName).split(System.lineSeparator());
+        int supplyCount = 0;
+        int buyCount = 0;
+        for (String data: arrayData) {
+            String[] separationData = data.split(",");
+            if (separationData[0].equals("buy")) {
+                buyCount += Integer.parseInt(separationData[1]);
```

Comment:
let's remove magic numbers and create constants for them, for example AMOUNT_INDEX